submission-scripts/script-templates/magpie-alluxio

```diff
@@ -0,0 +1,144 @@
+############################################################################
+# Alluxio Core Configurations
+############################################################################
+
+# Should Alluxio be run
+#
+# Specify yes or no. Defaults to no.
+#
+export ALLUXIO_SETUP=no
+
+# Alluxio Version
+#
+export ALLUXIO_VERSION="ALLUXIOVERSIONDEFAULT"
+
+# Path to your Alluxio build/binaries
+#
+# This should be accessible on all nodes in your allocation. Typically
+# this is in an NFS mount.
+#
+export ALLUXIO_HOME="ALLUXIODIRPREFIX/alluxio-${ALLUXIO_VERSION}"
+
+# ALLUXIO_WORKER_ON_MASTER
+#
+# Most big data projects do not have a "worker" that also
+# runs on the master node (such as Hadoop, Hbase, etc.). Alluxio is
+# an exception in which it often likes to have a worker running on the
+# master node for convenience of running tests and such.
+#
+# It is not required to run a worker on the master, but it can be
+# convenient. Specify the below to "yes" to run a worker on the
+# master.
+#
+export ALLUXIO_WORKER_ON_MASTER="yes"
+
+# Path to store data local to each cluster node, typically something
+# in /tmp. This will store local conf files and log files for your
+# job. If local scratch space is not available, consider using the
+# MAGPIE_NO_LOCAL_DIR_DIR option. See README for more details.
+#
+export ALLUXIO_LOCAL_DIR="LOCALDIRPREFIX/alluxio"
+
+# Path to store Alluxio Under File System data
+#
+# This should point to the path where you store your data
+# and have to be accessible on all nodes in your allocation.
+# Most likely it would be placed on lustre, like "/lustre/${USER}/alluxio"
+# but also it could be NFS mounted on all Alluxio nodes,
+# like "${ALLUXIO_HOME}/underFSStorage",
+# or ex. HDFS, like "hdfs://hdfsmaster:9000/alluxio/".
+# Check Alluxio documentation for more UnderFS storage options.
+#
+export ALLUXIO_UNDER_FS_DIR="LUSTREDIRPREFIX/alluxio"
+
+# ALLUXIO_DATA_CLEAR
+#
+# Alluxio has data files that are written to UnderFS and a journal file
+# containing metadata so the state of Alluxio filesystem is saved
+# and can be reused on multiple Alluxio runs.
+#
+# This flag enabled forces that data to be wiped which results
+# in a clean install of Alluxio every time Magpie runs it.
+#
+# Specify yes or no. Defaults to yes.
+export ALLUXIO_DATA_CLEAR="yes"
+
+# Memory capacity of each worker node
+#
+# By default, Alluxio allocates 2/3 of total system memory,
```

Comment:
as i mentioned above the default of 2/3 of total memory doesn't mesh with the fact there is a config setting here.  Should this setting be commented out by default?

Reply:
I responded in https://github.com/LLNL/magpie/pull/321#discussion_r487814977 let's continue the discussion there.